content/docs/guides/price-schedules.mdx

```diff
@@ -0,0 +1,102 @@
+
+
+## Create a Price Schedule
+
+First, lets create the price schedule that is to be associated with the Enterprise membership.  For this example, we'll use "USB Cord" as our Product.  
+
+Lets assume there is no tax or shipping costs, and the minimum order quantity is 1.  Our call to the OrderCloud API should like this (If successful, our call should return a `201 Created` response):
+
+```
+POST https://api.OrderCloud/v1/priceschedules
+Authentication: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9...
+Content-Type: application/json; charset=UTF-8
+
+{
+  "ID": "enterprise-priceschedule-id"
+  "Name": "USB Cord",
+  "MinQuantity": 1,
+  "PriceBreaks": [
+    {
+      "Quantity": 1,
+      "Price": 3.99
+    }
+  ]
+}
+```
+
+Next, lets make the exact same call, but change the `PriceBreaks` field to adjust for the **Startup** pricing.  The `PriceBreaks` array in your second call should look like this: 
+
+```json
+{
+  ...
+  "PriceBreaks": [
+    {
+      "Quantity": 1,
+      "Price": 5.99
+    }
+  ]
+}
+```
+
+
+
+## Create a Product / Price Schedule Relationship
+
+Now that we have our two price schedules created, the next step is to create two different product assignments based on party.  By making these product assignments, we are creating a special relationships between the product, a price schedule and different parties.  Lets first assign the enterprise USB Cord price schedule and the CloudTech 'Buyer' party that is subject to **Enterprise** pricing.  Both of these `POST` calls should garner `204 No Content` responses if successful.
+
+```
+POST https://api.OrderCloud/v1/products/assignments
+Authentication: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9...
+Content-Type: application/json; charset=UTF-8
+
+{
+  "ProductID": "usb-product-id",
+  "BuyerID": "cloudtech",
+  "PriceScheduleID": "enterprise-priceschedule-id"
+}
+```
+
+Now that our connection between the **Enterprise** USB Cord Price Schedule, the CloudTech 'Buyer' and the USB Cord product has been made, lets wire up our second assignment.
+
+```
+POST https://api.OrderCloud/v1/products/assignments HTTP/1.1
+Authentication: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9...
+Content-Type: application/json; charset=UTF-8
+
+{
+  "ProductID": "usb-product-id",
+  "BuyerID": "computerdudes",
+  "PriceScheduleID": "startup-priceschedule-id"
+}
+```
+
+Now that all of our assignments have been made, lets verify that we have successfully created special relationships between our two different buyers, two different price schedules and our product: USB Cord.
+
+In the context of Jane Doe from CloudTech, call `GET https://api.OrderCloud/v1/me/products/usb-product-id`, you should see the USB Cord product returned at the price of `"Price": 3.99`.
```

Comment:
might be over kill, but it might nice to add a link to something that helps a user to know which `context` they are in.  Do we have something on decoding the token with JWT?

Reply:
I added a little line in there - check it out @zeeshan and let me know what you think